examples/example_annotations/browse_annotations.py

```diff
@@ -0,0 +1,199 @@
+    def show_file(self, file_path_exp):
+        return file_path_exp[-3:] in ("csv", "tsv")
+
+    def csv_to_html(self, csv_file):
+
+        annotation_dict = {}
+        with open(self.ANNOTATION_FILE, 'r') as annotations:
+            for line in annotations.readlines():
+                key, value = line.strip().split('\t')
+                annotation_dict[key] = value
+
+        html = ""
+        # Open the CSV file for reading
+        mode = "rb" if py_version == 2 else "r"
+        with io.open(csv_file, mode) as csvfile:
+            if self.delimiter is None or self.quotechar is None:
+                try:
+                    dialect = csv.Sniffer().sniff(csvfile.read(2048))
+                    csvfile.seek(0)
+                except Exception as e:
+                    log.error("{} has exception {!s}".format(csv_file, e))
+                    return ""
+                reader = csv.reader(csvfile, dialect)
+            else:
+                reader = csv.reader(csvfile, delimiter=self.delimiter, quotechar=self.quotechar)
+
+            # initialize rownum variable
+            rownum = 0
+
+            html += "<table class='auto w3-table w3-bordered w3-border w3-hoverable w3-card-2 tablesorter'>\n"
+            folder, filename = split(csv_file)
+            href = join("file/", self.experiment_folder, filename)
+            html += "<caption><a href='%s'>%s</a></caption>" % (href, filename)
+
+            # generate table contents
+            for row in reader:  # Read a single row from the CSV file
+                if py_version == 2:
+                    row = [item.decode("utf8") for item in row]
+                # write header row. assumes first row in csv contains header
+                if rownum == 0:
+                    html += "<thead>"
+                    html += "<tr class='w3-green'>"  # write <tr> tag
+                    for column in row:
+                        html += '<th>' + column + '</th>'
+                    html += '<th>' + "Annotation" + '</th>'
+                    html += '</tr>\n'
+                    html += "</thead>"
+
+                # write all other rows
+                else:
+                    mdb_key = '\t'.join([row[k] for k in self.key])
+                    if mdb_key in annotation_dict:
+                        mdb_value = int(annotation_dict[mdb_key])
+                        html += '<tr id="tr_{}" bgcolor="#99ff99">'.format(mdb_key)
+                    else:
+                        mdb_value = 0
+                        html += '<tr id="tr_{}" bgcolor="#ffffff">'.format(mdb_key)
+                    print("mdb_value", mdb_value)
+                    for i_r, column in enumerate(row):
+                        html += '<td>' + column + '</td>'
+                    html += '<td><select id="{}" annotations_file="{}" onchange="annotate(this);">' \
+                            '<option value="0" {}>-</option>' \
+                            '<option value="1" {}>1: {}</option>' \
```

Comment:
How about using `OrderedDict` and `for` cycle?

Reply:
The idea was to changed  CsvToHtmlEditable in browse.py according to project needs. 
I didn't want to make the example complicated. 
But sure we can make it with for cycle and OrderedDict.

Reply:
Sounds reasonable.